.env.template

```diff
@@ -0,0 +1,4 @@
+KBC_TEST_TOKEN=
+# URL without trailing slash
```

Comment:
imo comment not necessary anymore because your rtrimming the trailing slash now below in the client constructor

Reply:
Yeah, well the client works without the slash, but the tests don't, that would need to make some assertions more benevolent - so yes, it is necessary at the moment